src/components/_shared/ControlledCheckbox/styles.module.scss

```diff
@@ -18,6 +20,11 @@
   margin-left: 12px;
 }
 
+.small {
```

Comment:
`.inputTitle` and .`small` have the same specificity, let's go with cascade order opposed to the `!important` statement.